### DISCUSSION TITLE
Upload failed: Certificate is not valid (Cannot read properties of null (reading '1'))

### DISCUSSION THREAD

**Discussion by shedowe19** [ZoeyVid/NPMplus#841] [Q&A] [answered]
I become the Error of Certbot Certs and Custom Certs when i import the Cert...

Please Help me.

**Comment by shedowe19** [comment]
:81/api/nginx/certificates/validate:1 
        
        
       Failed to load resource: the server responded with a status of 400 ()

**Comment by shedowe19** [comment]
{"error":{"code":403,"message":"Permission Denied"},"debug":{"stack":["PermissionError: Permission Denied","    at /app/lib/access.js:295:13","    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"],"previous":{"name":"PermissionError","message":"Permission Denied","public":true,"status":403,"permission":"certificates:list"}}}

**Comment by shedowe19** [comment]
zoeyvid/npmplus:400

By this Verson it Funtion but not on the Newest 401 Version.

**Reply by Zoey2936** [reply]
not sure, could be related to python/nodejs/liquidjs or @apidevtools/json-schema-ref-parser updates, I will check it in afew days

**Reply by Zoey2936** [reply]
should be fixed now

**Comment by Matthias-vdE** [comment]
Just letting you know that I have the same problem when uploading existing, or requesting new, certificates. Version 400 did not have the issue, version 401 does.

**Reply by Zoey2936** [reply]
can you try the :develop tag?

**Reply by Matthias-vdE** [reply]
Hi, thanks for the quick answer.

Unfortunately, re-deploying the stack with the :develop tag makes the issue return. Error message "Upload failed: Certificate is not valid (Cannot read properties of null (reading '1'))" (But nothing in the docker container log).

**Reply by Zoey2936** [reply]
that looks like a diffrent issue

**Reply by Zoey2936** [reply]
but can confirm:

https://www.openssl.org/news/cl33.txt:
> ### Changes between 3.2 and 3.3.0 [9 Apr 2024]
> * The `-verify` option to the `openssl crl` and `openssl req` will make
 >  the program exit with 1 on failure.

**Reply by Matthias-vdE** [reply]
Noted. Will stay on release 400 for the time being :)

**Reply by Zoey2936** [reply]
should be fixed now

**Comment by Zoey2936** [comment, marked as answer]
see #853

**Reply by Zoey2936** [reply]
should be fixed now
